console/console-init/ui/src/modules/project-detail/components/AccessCredentials/AccessCredentials.tsx

```diff
@@ -82,164 +75,177 @@ const AccessCredentials: React.FunctionComponent<IAccessCredentialsProps> = ({
-        />
-      )}
-      <br />
-      {messaging.commandAddressName && (
-        <InputText
-          label={"Command address name"}
-          type={"text"}
-          value={getCombinedString(
-            messaging.commandAddressName.type,
-            messaging.commandAddressName.plan
-          )}
-          isReadOnly={true}
-          enableCopy={true}
-          isExpandable={messaging.commandAddressName.plan ? true : false}
-          ariaLabel={"input-messaging-commandAddressName"}
-        />
-      )}
-      <br />
-    </>
-  );
+
+  const Messaging = () => {
+    const {
+      url,
+      username,
+      password,
+      addressSpace,
+      commandAddressName,
+      eventsAddressName,
+      telemetryAddressName
+    } = messaging && messaging;
+    return (
+      <>
+        <Text component={TextVariants.h2}>Messaging</Text>
+        <br />
+        {url && (
+          <InputText
+            label={"Tenant Id"}
+            type={"text"}
+            value={url}
+            isReadOnly={true}
+            enableCopy={true}
+            ariaLabel={"input-messaging-url"}
+          />
+        )}
+        <br />
+        {username && (
+          <InputText
+            label={"User name"}
+            type={"text"}
+            value={username}
+            isReadOnly={true}
+            enableCopy={true}
+            ariaLabel={"input-messaging-username"}
+          />
+        )}
+        <br />
+        {password && (
+          <InputText
+            label={"Password"}
+            type={"password"}
+            value={password}
+            isReadOnly={true}
+            ariaLabel={"input-messaging-password"}
+          />
+        )}
+        <br />
+        {addressSpace && (
+          <InputText
+            label={"Address Space"}
+            type={"text"}
+            value={addressSpace}
+            isReadOnly={true}
+            enableCopy={true}
+            ariaLabel={"input-messaging-addressSpace"}
+          />
+        )}
+        <br />
+        {eventsAddressName && (
+          <InputText
+            label={"Events address name"}
+            type={"text"}
+            value={getCombinedString(
+              eventsAddressName.type,
+              eventsAddressName.plan
+            )}
+            isReadOnly={true}
+            enableCopy={true}
+            isExpandable={eventsAddressName.plan ? true : false}
+            ariaLabel={"input-messaging-eventsAddressName"}
+          />
+        )}
+        <br />
+        {telemetryAddressName && (
+          <InputText
+            label={"Telemetry address name"}
+            type={"text"}
+            value={getCombinedString(
+              telemetryAddressName.type,
+              telemetryAddressName.plan
+            )}
+            isReadOnly={true}
+            enableCopy={true}
+            isExpandable={telemetryAddressName.plan ? true : false}
+            ariaLabel={"input-messaging-telemetryAddressName"}
```

Comment:
Purpose of ariaLabel is for web accessibility which will announce by software for blind or color contrast person.
Always pass meaningful message which tell about the specific item.  don't pass something id including -